rust.yaml

```diff
@@ -22,7 +23,6 @@ environment:
       - busybox
       - ca-certificates-bundle
       - build-base
-      - rust-stage0
```

Comment:
Don't we still need this?

Reply:
Yeah, it was actually duplicated. See line 38/39.

Reply:
That just means we _really_ need it.